openqa_bugfetcher/issues/bugzilla_gnome_issue.py

```diff
@@ -1,35 +1,6 @@
-#!/usr/bin/python3
+from openqa_bugfetcher.issues import BugzillaBaseIssue
 
-from openqa_bugfetcher.issues import BaseIssue
-from collections import OrderedDict
-import requests
-import json
 
-
-class BugzillaGnomeIssue(BaseIssue):
+class BugzillaGnomeIssue(BugzillaBaseIssue):
     prefixes = {"bgo"}
-
-    def fetch(self, conf):
-        def json_rpc_get(url, method, params):
-            get_params = OrderedDict({"method": method, "params": json.dumps([params])})
-            return requests.get(url, params=get_params)
-
-        url = "https://bugzilla.gnome.org/jsonrpc.cgi"
```

Comment:
I think you forgot to set the url accordingly for each bugzilla subclass.

Reply:
oh, thx